apps/prairielearn/src/lib/workspaceHost.sql

```diff
@@ -1,4 +1,4 @@
--- BLOCK set_host_unhealthy
+-- BLOCK set_hosts_unhealthy
```

Comment:
This is almost a verbatim copy of the `workspace_hosts_all_unhealthy` admin query. If we were willing to give up the raw admin query, we could use the `markAllWorkspaceHostsUnhealthy` functionality behind a "Refresh workspace hosts" button on the admin "workspaces" page?

Reply:
I'm certainly fine to get rid of the admin query. I would even be fine with not having any button to do it. On the very rare occasion that we want to do this outside of deploys (I'm not sure this ever happens) then we can always run the `--refresh-workspace-hosts` command.

Reply:
Removed in 27f5b2b.